mods/default/functions.lua

```diff
@@ -715,9 +715,40 @@ function default.register_craft_metadata_copy(ingredient, result)
 	end)
 end
 
+--
+-- Log API / helpers
+--
+
+local non_player_action_log = minetest.settings:get_bool("enable_non_player_action_log") ~= false
+
+function default.log_player_action(player, pos, message)
+	local who = player:get_player_name()
+	if not player.is_fake_player and player:is_player() then
+		-- log action of real player
+		minetest.log("action",  who .. " " .. message .. " at " .. minetest.pos_to_string(pos))
+	elseif non_player_action_log then
+		-- log action of non real player
+		who = who .. "(" .. (type(player.is_fake_player) == "string"
```

Comment:
I can guess what this does but it's not documented...

Reply:
gane_api.txt updated: https://github.com/minetest/minetest_game/pull/2941/commits/26553bfaee20f92deadcb748942010a2ea890ef8